cli/get.go

```diff
@@ -128,16 +128,19 @@ A role must be specified when using this command through the --role flag. You ma
 		}
 
 		account.MostRecentRole = roleName
-
-		switch outputType {
-		case outputTypeEnvironmentVariable:
-			credentials.WriteFormat(os.Stdout, shell)
-			return nil
-		case outputTypeAWSCredentialsFile:
-			acc := Account{ID: args[0], Name: args[0]}
-			newCliEntry := NewAWSCliEntry(&credentials, &acc)
-			return SaveAWSCredentialInCLI(awsCliPath, newCliEntry)
-		default:
-			return fmt.Errorf("%s is an invalid output type", outputType)
-		}
+		return echoCredentials(args[0], args[0], credentials, outputType)
```

Comment:
I see the old code uses `args[0]`

```
acc := Account{ID: args[0], Name: args[0]}
```

But should we use `applicationID` and `label` here?

```suggestion
		return echoCredentials(applicationID, label, credentials, outputType)
```

Reply:
Yes

Reply:
Perhaps, but I don't want to make that change because that's not what was done originally and I don't know if it will break things. We can address in a separate PR

Reply:
Makes sense.